org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/validation/IssueCodes.java

```diff
@@ -20,4 +20,6 @@
 
 	String MISSING_TYPE = PREFIX + "MissingType";
 
+	String DUPLICATE_PROPETY = PREFIX + "duplicate_property";
```

Comment:
Why did you decide to use different casing than above?

Reply:
Fixed.